uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/manager-leave/components/manager-leave-requests.js

```diff
@@ -491,15 +497,20 @@ define([
      * Register events which will be called by other modules
      */
     function registerEvents() {
-      $rootScope.$on('LeaveRequest::updatedByManager', function () {
-        leaveRequestAPICache = false;
-        vm.refresh();
-      });
+      $rootScope.$on('LeaveRequest::updatedByManager', refreshFromServer);
+      $rootScope.$on('LeaveRequest::new', refreshFromServer);
+    }
 
-      $rootScope.$on('LeaveRequest::new', function () {
-        leaveRequestAPICache = false;
-        vm.refresh();
-      });
+    /**
+     * Refresh leave requests from server
+     */
+    function refreshFromServer() {
```

Comment:
`refreshFromServer` is confusing as refresh should happen always from server. Perhaps a better name like `refreshWithoutCache` is more meaningful.